src/topp/DatabaseSuitability.cpp

```diff
@@ -162,26 +161,20 @@ class DatabaseSuitability :
     String in_spec = getStringOption_("in_spec");
     String in_novo = getStringOption_("in_novo");
     String out = getStringOption_("out");
-    double novo_fract = getDoubleOption_("novor_fract");
+    double cut_off_fract = getDoubleOption_("cut_off_fract");
     double FDR = getDoubleOption_("FDR");
     bool no_re_rank = getFlag_("force_no_re_rank");
```

Comment:
stay with `no_rerank` in the class (and also the TOPP tool by using a subsection).
Replace all occurrences of `force_no_re_rank` in the PR.